build.gradle

```diff
@@ -264,8 +353,29 @@ project(':par-connector') {
             } catch (e) {
             }
             schedProcess.destroy();
+            // workaround for killing PARServe on windows
+            if (isWindows) {
```

Comment:
Well, I didn't run the testsuite on a linux system, only tested the PARserve addon manually on a scheduler. So maybe you're right and I need to manually kill the Rserve process in the testsuite for linux